src/main/java/seedu/address/logic/parser/InProgressCommandParser.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.InProgressCommand;
+import seedu.address.logic.commands.TodoCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.event.Description;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.EventName;
+import seedu.address.model.event.EventStatus;
+
+public class InProgressCommandParser implements Parser<InProgressCommand> {
+    private static final EventStatus TODO_EVENT_STATUS = EventStatus.IN_PROGRESS;
```

Comment:
You might want to consider changing the name of this to IN_PROGRESS_EVENT_STATUS